docs/src/main/sphinx/release/release-358.md

```diff
@@ -0,0 +1,40 @@
+# Release 358 (XX May 2021)
+
+## General
+
+* Support any query in {doc}`/sql/show-stats`. ({issue}`8026`)
+* Improve performance of complex queries involving joins and `TABLESAMPLE`. ({issue}`8094`)
+* Improve performance of `ORDER BY ... LIMIT` queries on sorted data. ({issue}`6634`)
+* Reduce worker graceful shutdown time. ({issue}`8149`)
+* Fix query failure when one of join build side's columns is of non-orderable type. ({issue}`7723`)
+* Fix failure for queries containing repeated ordinals in `GROUP BY` clause.
+  Example: `SELECT x FROM t GROUP BY 1, 1`. ({issue}`8023`)
+* Fix failure for queries containing repeated expressions in `ORDER BY` clause of an aggregate function.
+  Example: `SELECT count(x ORDER BY y, y) FROM (VALUES ('a', 2)) t(x, y)`. ({issue}`8080`)  
+
+## JDBC Driver
+* Remove legacy driver classes `com.facebook.presto.jdbc.PrestoDriver`
+  and `io.prestosql.jdbc.PrestoDriver`. ({issue}`8042`)
+* Remove legacy JDBC URL prefix `jdbc:presto:`. ({issue}`8042`)
+
+## Hive connector
+
+* Add support for reading from Hive views which use `LATERAL VIEW EXPLODE`
+  or `LATERAL VIEW OUTER EXPLODE` over array of `STRUCTs`. ({issue}`8120`)
+  
+## Iceberg connector
+
+* Fix failure when listing materialized views in `information_schema.tables` and `system.jdbc.tables` ({issue}`8151`)
+
+## Memory connector
+
+* Improve performance of certain complex queries involving joins. ({issue}`8095`)
+
+## SPI
+
+* Remove deprecated ``ConnectorPageSourceProvider#createPageSource`` method overrides. ({issue}`8077`)
```

Comment:
We should be consistent about this (and fix existing release notes), but I think it looks nicer to write these as `ConnectorPageSourceProvider.createPageSource()`. This better matches The Java Tutorials, etc. I don't feel strongly, but we should pick a style and document it in https://github.com/trinodb/trino/wiki/Release-Notes-Guidelines